src/scanner.l

```diff
@@ -2315,6 +2310,18 @@ NONLopt [^\n]*
                                                 }
                                                 yyextra->current->name= yyextra->current->name.mid(7);
                                               }
+                                              else if (yyextra->current->name.left(10)=="cconstexpr ")
```

Comment:
I think there is a typo here (extra `c` before `constexpr`), right?

Reply:
100% right, fixed